catboost/python-package/catboost/core.py

```diff
@@ -1215,18 +1215,30 @@ def fit(self, X, y=None, cat_features=None, pairs=None, sample_weight=None, grou
                          save_snapshot, snapshot_file, snapshot_interval)
 
     def _predict(self, data, prediction_type, ntree_start, ntree_end, thread_count, verbose):
+        single_flag = False
```

Comment:
For consistency, please apply the changes from this pull request to the following methods.

- CatBoost.staged_predict()
- CatBoostClassifier.predict()
- CatBoostClassifier.staged_predict()
- CatBoostClassifier.predict_proba()
- CatBoostClassifier.staged_predict_proba()
- CatBoostRegressor.predict()
- CatBoostRegressor.staged_predict()